front/src/components/App/MaintenanceArea/AuctionSitesList.tsx

```diff
@@ -0,0 +1,31 @@
+import React from "react";
+import {deleteAucitonSite} from "./Utils";
+import {AuctionSite} from "./AuctionSitesManagement";
+import {AuctionSitesInterface} from './AuctionSitesAddition'
+
+const AuctionSitesList = (auctionInterface: AuctionSitesInterface) => {
+
+  const handleDeletion = (e: React.MouseEvent<HTMLButtonElement>) => {
+    e.preventDefault();
+    auctionInterface.setAuctionsSites([...deleteAucitonSite(auctionInterface.auctionsSites, e.currentTarget.value)])
```

Comment:
tutaj nie wiem co się dzieje. Koniecznie trzeba to rozbić by poprawić czytelność